src/client.rs

```diff
@@ -81,38 +82,35 @@ impl Client {
                 .send()
                 .await;
 
-            match bucket_location {
-                Ok(location) => {
-                    if let Some(constraint) = location.location_constraint() {
-                        let constraint_str = match constraint.as_str() {
-                            // For us-east-1, `location.location_constraint() returns Some<Unknown<UnknownVariantValue("")>>, with it's `as_str()` returning "".
-                            // We explicitly handle this case by mapping it to "us-east-1".
-                            "" => "us-east-1",
-                            other => other,
-                        };
-
-                        if constraint_str == self.region {
-                            buckets_in_region.push(bucket);
-                        }
-                    }
-                }
-                Err(e) => {
-                    return Err(AppError::new(
-                        format!("Failed to get bucket location for bucket {}", bucket.name),
-                        e,
-                    ));
+            let location = bucket_location.map_err(|e| {
+                AppError::new(
+                    format!("Failed to get bucket location for bucket {}", bucket.name),
+                    e,
```

Comment:
I assume this will end up in `.stu/error.log` as well as showing the message to the user?

Reply:
Yes, it will be logged when we notify with `AppError`.